provider/rancher.go

```diff
@@ -44,6 +48,10 @@ type rancherData struct {
 	Health     string
 }
 
+func init() {
+	withoutPagination.Filters["limit"] = 0
```

Comment:
You're accessing the fields of a pointer to a script. Wouldn't that cause a nil dereference error unless you initialized the pointer?